ee/tabby-ui/package.json

```diff
@@ -54,21 +56,26 @@
     "clsx": "^1.2.1",
     "compare-versions": "^6.1.0",
     "copy-to-clipboard": "^3.3.3",
+    "date-fns": "^3.6.0",
     "downshift": "^8.2.2",
     "filename2prism": "^3.0.0",
     "focus-trap-react": "^10.1.1",
     "graphql": "^16.8.1",
     "humanize-duration": "^3.31.0",
     "jwt-decode": "^4.0.0",
     "lodash-es": "^4.17.21",
+    "lucide-react": "^0.365.0",
```

Comment:
```suggestion
```

Embed icons svg individually in `icons.tsx` to make consistent properties.

Reply:
This was added by `npx shadcn-ui@latest add calendar`, I keep it because I didn't wanna change the UI codes added by shadcn
Now I replace `lucide-react` with `icons.tsx` in the calendar component

Reply:
ic in that case, you might leave it as is.